### ISSUE TITLE
db_url path does not exist, even after initialization

### ISSUE THREAD

**Issue by HelionSmoker** [sysid/bkmr#3]
## Tried
```
$ bkmr create-db ~dt/bmarks.db
Show not implemented yet. "/home/link/.local/share/bmarks.db"
Creating database at "/home/link/.local/share/bmarks.db"
Database created at "/home/link/.local/share/bmarks.db"

$ bkmr add "https://github.com/sysid/bkmr"
Error: db_url path does not exist: "../db/bkmr.db"
```

The file was indeed generated at `~dt/bmarks.db`, but the program doesn't seem to notice it. If i create `../db/bkmr.db` exactly, then it does work. Any thoughts on why this happens?

## Info
Version: bkmr 0.2.1
Source: `cargo install --git https://github.com/sysid/bkmr`

**Comment by sysid** [comment]
@HelionSmoker thanks for pointing this out. First quick check:
1) you have make the path to your URL bookmark database available. See https://github.com/sysid/bkmr#configuration
2) the "not implemented yet" messages seems to be a bug. Will look into it.
Let me know whether setting the environment variable does solve your problem.

**Comment by HelionSmoker** [comment]
Yes, everything seems to work find after setting the environment variable.

I think it was just muscle memory, since I haven't seen programs that required the database path to be declared both for initialization and usage. Couldn't at that point the program just read the env variable, skipping any argument for `db-create`?

**Comment by sysid** [comment]
@HelionSmoker thanks for the hint. Might be a nice improvement in one of the next releases.

**Comment by sysid** [comment]
Hi @HelionSmoker , if you like bkmr you can give it a vote on[ "crate of the week"](https://users.rust-lang.org/t/crate-of-the-week/2704/1158).
